src/parsers/manifestjson.js

```diff
@@ -1149,6 +1149,10 @@ export default class ManifestJSONParser extends JSONParser {
   }
 
   validateCspPolicyString(policy, manifestPropName) {
+    if (typeof policy !== 'string') {
```

Comment:
@willdurand I wonder if we should also tweak the caller (`validateCspPolicy') to only iterate over `Object.keys(policy)` if policy is not null and not an array (given that for the other primitive types but array, Object.keys would already be an empty array and not doing anything in practice), the tweak in this PR should already prevent the kind of unexpected issues we may be hitting but in the end if the csp value is an array then it is already invalid on its own and validating each element of the array not strictly necessary anymore.

Reply:
I added a test case to cover `null`.